test/annotation_layer_builder_overrides.css

```
@@ -29,14 +29,20 @@
   -webkit-appearance: none;
 }
 
-.annotationLayer :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton) > a,
+.annotationLayer
+  :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton):not(.hasBorder)
+  > a,
 .annotationLayer .popupTriggerArea::after,
 .annotationLayer .fileAttachmentAnnotation .popupTriggerArea {
   opacity: 0.2;
   background: rgba(255, 255, 0, 1);
   box-shadow: 0 2px 10px rgba(255, 255, 0, 1);
 }
 
+.annotationLayer .linkAnnotation.hasBorder {
+  background-color: rgba(55, 255, 0, 0.2);
```

Is this a typo, or actually intended?
Looking at the other CSS file, I'd sort of expect the following instead.
```suggestion
  background-color: rgba(255, 255, 0, 0.2);
```